.github/workflows/ci.yml

```diff
@@ -29,6 +29,16 @@ jobs:
           - 7.2
           - 7.3
           - 7.4
+  typoscript-lint:
+    name: "TypoScript linter"
+    runs-on: ubuntu-20.04
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v2
+      - name: "Run TypoScript lint"
+        uses: TYPO3-Continuous-Integration/TYPO3-CI-Typoscript-Lint@v1
+    strategy:
+      fail-fast: false
```

Comment:
This is not needed in this case it this setting does not make any difference for jobs without a matrix.